### PR TITLE
Wrote React.js Conf Round-up 2015

### PR DESCRIPTION
**Pull request by steveluscher**
conf-roundup-2015

### CODE REVIEW

File: docs/_posts/2015-02-18-react-conf-roundup-2015.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
+---
+title: React.js Conf Round-up 2015
+layout: post
+author: Steven Luscher
+---
+
+It was a privilege to welcome the React community to Facebook on January 28–29 for the first-ever React.js Conf, and a pleasure to be be able to unveil three new technologies that we've been using internally for quite some time: GraphQL, Relay, and React Native.
```
Comment (spicyj) [inline docs/_posts/2015-02-18-react-conf-roundup-2015.md]: maybe "internally at Facebook"

**Comment by spicyj** [comment]
I might leave the talks in chronological order instead of bumping GraphQL/Relay up, but lgtm either way.

**Comment by zpao** [comment]
If you keep this against `0.12-stable`, just make sure it gets cherry-picked over to master so we don't lose it when we make the next branch cut.

File: docs/_posts/2015-02-18-react-conf-roundup-2015.md (hunk 1 above)
Comment (steveluscher) [inline docs/_posts/2015-02-18-react-conf-roundup-2015.md]: Do you mean to clarify that it was _physically_ at Facebook? I'll change it to “Facebook HQ” – that should do it.

Comment (spicyj) [inline docs/_posts/2015-02-18-react-conf-roundup-2015.md]: Sorry, I meant instead of "we've been using internally for quite some time".

**Comment by steveluscher** [comment]
Rolled in @spicyj's feedback.

**Pull request merged**